google-cloud-compute-v1/.repo-metadata.json

```diff
@@ -4,11 +4,11 @@
     "distribution_name": "google-cloud-compute-v1",
     "language": "ruby",
     "name": "compute",
-    "name_pretty": "Google Cloud Compute V1 API",
+    "name_pretty": "Google Cloud Compute V1 (BETA) API",
     "product_documentation": "https://cloud.google.com/compute/",
     "repo": "googleapis/google-cloud-ruby",
     "requires_billing": true,
-    "ruby-cloud-description": "google-cloud-compute-v1 is the official client library for the Google Cloud Compute V1 API.",
+    "ruby-cloud-description": "google-cloud-compute-v1 is the official client library for the Google Cloud Compute V1 API. This library is considered to be in beta. This means while stable it is still a work-in-progress and under active development, and might get backwards-incompatible changes at any time.",
```

Comment:
Just a reminder to get these changes added to the configuration in the BUILD.bazel file, so that the next autosynth run doesn't revert them.

Reply:
https://github.com/googleapis/googleapis-discovery/pull/64

Reply:
done